test/jobs/test_samtools.py

```diff
@@ -0,0 +1,111 @@
+import re
+import os
+
+from .. import set_config
+from ..factories.alignment_input import create_cram_input
+from ..factories.batch import create_local_batch
+from ..factories.config import PipelineConfig, WorkflowConfig
+from ..factories.sequencing_group import create_sequencing_group
+from .helpers import get_command_str
+
+from cpg_workflows.jobs.samtools import samtools_stats
+from cpg_utils import Path
+from cpg_workflows.filetypes import CramPath
+from cpg_utils.hail_batch import image_path
+
+
+def default_config() -> PipelineConfig:
+    return PipelineConfig(
+        workflow=WorkflowConfig(
+            dataset='samtools-test',
+            access_level='test',
+            sequencing_type='genome',
+            check_inputs=False,
+        ),
+        images={
+            'samtools': 'test_image',
+        },
+        other={
+            'references': {
+                'broad': {
+                    'ref_fasta': 'hg38_reference.fa',
+                    'dragmap_prefix': 'gs://a-cpg-bucket/dragen_reference/',
+                }
+            },
+        },
+    )
+
+
+class TestSamtoolsRun:
+    def test_create_cram_input_method_creates_inputs(self, tmp_path: Path):
```

Comment:
Although a good idea  to test the `create_cram_input` factory, these tests should only be testing the samtools job module. Factory tests will live in their own test files and will be done at a later date.